### PR TITLE
Updating SDK dependency

### PR DESCRIPTION
**Pull request by erikbosch**
Upgrading dependencies to fix vulnerabilities.
I see no real need to have explicit dependencies for many of the packages, as we do not intend to use them directly from the app, but rather indirectly through the SDK, or?

### CODE REVIEW

File: app/requirements.in [hunk 1]
```diff
@@ -12,8 +12,4 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-grpcio==1.59.0
-protobuf==4.24.4
-cloudevents==1.10.0
-aiohttp == 3.9.3
-velocitas-sdk==0.14.1
+velocitas-sdk==0.15.1
```
Comment (dennismeister93) [inline app/requirements.in:15]: lets wait for 0.15.2

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: app/requirements.in (hunk 1 above)
Comment (erikbosch) [reply app/requirements.in:15]: Agree,

Comment (erikbosch) [reply app/requirements.in]: Updated

**Review by doosuu — APPROVED** [review]
lgtm

**Comment by erikbosch** [comment]
I would say this one is replaced by #257, closing it

**Pull request closed without merging**